### PR TITLE
Add new actual app + theme implementation

### PR DESCRIPTION
**Pull request by KlaKalma**
- Implement the app framework with android studio
- Change gitignore to handle local folders
- Add the theme (for now only dark mode)

### CODE REVIEW

**Review by joriba — APPROVED** [review]
lgtm

File: app/src/main/java/ch/epfl/cs311/wanderwave/ui/App.kt
Comment (joriba) [inline app/src/main/java/ch/epfl/cs311/wanderwave/ui/App.kt:1]: Incorrect formatting (4 spaces)

File: app/src/main/java/ch/epfl/cs311/wanderwave/ui/theme/Type.kt
Comment (joriba) [inline app/src/main/java/ch/epfl/cs311/wanderwave/ui/theme/Type.kt:1]: Here too, 4 spaces

File: gradle/wrapper/gradle-wrapper.properties
Comment (joriba) [inline gradle/wrapper/gradle-wrapper.properties:1]: Carriage returns - do we care about consistency there?

**Review by joriba — CHANGES_REQUESTED** [review]
run ktfmt to update indentation to 4 spaces consistently in all kotlin files

**Review by joriba — APPROVED** [review]
Seems good now after ktfmt run

**Review by KlaKalma — COMMENTED** [review]
Lgtm

**Pull request merged**